Sparkle/SUAppcastItem.m

```diff
@@ -55,6 +55,11 @@ - (BOOL)isCriticalUpdate
     return [self.propertiesDictionary[SUAppcastElementTags] containsObject:SUAppcastElementCriticalUpdate];
 }
 
+- (BOOL)isInformationOnlyUpdate
+{
+	return (([self infoURL] != nil) && ([self fileURL] == nil));
```

Comment:
For methods returning BOOL, this can be quite a gotcha, true (I usually use `BOOL b = !!ptr`). However this is a lot more readable. At the very least, remove the ( ) and [ ] if you want to keep the == nil